src/cljs/athens/electron.cljs

```diff
@@ -315,6 +317,42 @@
             mtime (or mtime1 (.. fs (statSync filepath) -mtime))]
         (assoc db :db/mtime mtime))))
 
+  (reg-event-fx
+    :local-storage/set-rfdb
```

Comment:
Do you think `:local-storage/set-rfdb` should just replace `:init-rfdb`?

Reply:
Yeah now that I think about this my first implementation for this can be seen below, but in the first iteration, I did not (de)serialize `theme/dark` or `appearance/width` hence the UI misbehaved and I thought I was doing something wrong therefore to debug by printing values in the console log I took the re-frame approach. But I think following would be the better solution for setting `theme/dark` and other such values from local storage. What do you think @tangjeff0 ?
```
 :theme/dark    (or (js/localStorage.getItem (cljs.reader/read-string "theme/dark"))
                     false)
```